sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -82,6 +105,16 @@ object AlluxioUtils {
         // replace all of rootPaths which are already unique
         val rootPaths = relation.location.rootPaths.map(replaceFunc)
 
+        // check the alluxio paths in root paths exist or not
+        // throw out an exception to stop the job when any of them is not mounted
+        rootPaths.foreach(rootPath => {
+          val pathStr = rootPath.toString
+          val matchedSet = replaceMap.values.filter(reg => pathStr.startsWith(reg))
+          if (matchedSet.size == 1) {
```

Comment:
What about  `matchedSet.size != 1`, no need to check ?
If you just want to know whether the pathStr starts with a matched string in `replaceMap.values`, you can simply try
```
replaceMap.values.find(reg => pathStr.startsWith(reg)).foreach{ matched => 
  checkAlluxioMounted(relation.sparkSession, matched)
}
```

Reply:
Good idea. Updated to your simple way.